Changelog.md

```diff
@@ -4,19 +4,21 @@ Language Features:
 
 
 Compiler Features:
+* Commandline Interface: Return exit code ``2`` on uncaught exceptions.
 * Commandline Interface: Add `--no-cbor-metadata` that skips CBOR metadata from getting appended at the end of the bytecode.
 * Standard JSON: Add a boolean field `settings.metadata.appendCBOR` that skips CBOR metadata from getting appended at the end of the bytecode.
 * Yul Optimizer: Allow replacing the previously hard-coded cleanup sequence by specifying custom steps after a colon delimiter (``:``) in the sequence string.
 * Language Server: Add basic document hover support.
 
 
 Bugfixes:
+* Solidity Upgrade Tool ``solidity-upgrade``: Return exit code ``2`` on uncaught exceptions.
 
 
 ### 0.8.17 (2022-09-08)
 
 Important Bugfixes:
- * Yul Optimizer: Prevent the incorrect removal of storage writes before calls to Yul functions that conditionally terminate the external EVM call.
+* Yul Optimizer: Prevent the incorrect removal of storage writes before calls to Yul functions that conditionally terminate the external EVM call.
```

Comment:
There should actually be a space there. Not sure why, but it's like this in the whole changelog so let's not change it now. And please leave the the `Compiler Features` section as is for now. I'll fix it when I'll be sorting the changelog for the release.